package.json

```diff
@@ -10,7 +10,7 @@
     "start": "NODE_ENV=production node server.js",
     "format-staged": "pretty-quick --staged --no-restage --bail",
     "format-check": "pretty-quick --check --verbose",
-    "lint": "eslint src"
+    "lint": "eslint src pages"
```

Comment:
I still need to make one for the pre-commit hook using https://github.com/okonet/lint-staged to check only staged files, and use this one for the CI checks.

Reply:
OK, this was applied per https://github.com/iterative/dvc.org/issues/716#issuecomment-549943912.